README.md

```diff
@@ -94,6 +94,25 @@ r.word_of_the_day()
     # Output: {"word": "qualtagh", "definations": [{"text": "The first person one encounters, either after leaving one\'s home or (sometimes) outside one\'s home, especially on New Year\'s Day.", "source": "wiktionary", "partOfSpeech": "noun"}, {"text": "A Christmas or New Year\'s ceremony, in the Isle of Man; one who takes part in the ceremony. See the first extract.", "source": "century", "partOfSpeech": "noun"}]}
     ```
 
+## Development
+
+ Assuming that you have [`Python`](https://www.python.org/downloads/release/python-370/) and [`pipenv`](https://docs.pipenv.org) installed, set up your environment and install the required dependencies like this instead of the `pip install random-word` defined above:
```

Comment:
Change link of Python w/ https://www.python.org/

Reply:
I was busy in some tasks at my end, will do it tomorrow and Have you looked at the error message I sent you? @vaibhavsingh97

Reply:
@igagankalra Will review it, I haven't reviewed